tests/epyccel/test_arrays_multiple_assignments.py

```diff
@@ -140,11 +140,14 @@ def f():
 
     # Check that we got exactly 1 Pyccel error
     assert errors.has_errors()
-    assert errors.num_messages() == 1
+    assert errors.num_messages() == 2
```

Comment:
The comment line should be updated: "1 Pyccel error" -> "2 Pyccel errors"

Reply:
Fixed [130f7ca](https://github.com/pyccel/pyccel/pull/993/commits/130f7ca137368cbf5575facac9c4f1c28a22a099)